client/inner_client.go

```diff
@@ -71,15 +71,8 @@ func (c *innerClient) setServiceMode(newMode pdpb.ServiceMode) {
 	if newMode == c.serviceMode {
 		return
 	}
-	log.Info("[pd] changing service mode",
-		zap.String("old-mode", c.serviceMode.String()),
-		zap.String("new-mode", newMode.String()))
 	c.resetTSOClientLocked(newMode)
-	oldMode := c.serviceMode
 	c.serviceMode = newMode
-	log.Info("[pd] service mode changed",
```

Comment:
Do we need to keep one of the two logs?

Reply:
```
if newMode != pdpb.ServiceMode_PD_SVC_MODE {
log.Info("[pd] changing from non-microservice to microservice")
}else{
log.Info("[pd] changing from microservice to non-microservice")
}
```

Reply:
how about it?

Reply:
Is it similar to line 87 and line 91?

Reply:
resetTSOClientLocked and setServiceMode may be called at different places?

Reply:
done